modules/exploits/windows/smb/ms17_010_eternalblue_win8.py

```diff
@@ -54,7 +54,7 @@
         - The idea is from "Bypassing Windows 10 kernel ASLR (remote) by Stefan Le Berre" (see link in reference)
         - The exploit is also the same but we need to trigger bug twice
         - First trigger, set MDL.MappedSystemVa to target pte address
-          - Write '\x00' to disable the NX flag
```

Comment:
Looks like this was copied directly from the original docstring, but the fact that it was added to `metadata` sent it over JSON-RPC.

Reply:
Or, rather, it sent nothing (null).